XdmodAID.md

```diff
@@ -0,0 +1,853 @@
+#Introduction
+
+Xdmod is currently developed and used in the HPC community as a way to show
+system usage for an HPC cluster back to uses.  They have build support for
+OpenStack into it, but not OpenShift.  It holds the promise of being a unified
+platform for consoldating HPC and Cloud services.
+
+As it is currently being used for billing and showback on a HPC cluster
+at Harvard, and it initally seemed to be a turnkey type system, we started
+to set this it up on an openshift cluster.  Unfortunately, we have found that it
```

Comment:
remove up

Reply:
I had to read this twice to figure out what you meant.  I decided that you mean "remove 'up'" as opposed to removing more.